tests/components/application-connector/README.md

```diff
@@ -20,17 +21,64 @@ These are the component tests for Application Connector.
 
 The tests consist of:
 - [Application CRs](./resources/charts/gateway-test/templates/applications/) describing the test cases
+- [Secrets](./resources/charts/gateway-test/templates/applications/credentials) referenced by the Application CRs
 - [Test runners](./test/application-gateway/) with various check for subsets of cases, grouped by the Application CRs
 - [Mock application](./tools/external-api-mock-app/) which simulates the remote endpoints
 
 Additionally, the following resources are created on the cluster:
 - [Service Account](./resources/charts/gateway-test/templates/service-account.yml#L2), used by the tests to read the Application CRs
+- [Secrets](./resources/charts/gateway-test/templates/target-api-mock/credentials) used by the Mock application to configure mTLS servers 
 
 The tests are executed as a Kubernetes Job on a Kyma cluster, where the tested Application Gateway is installed. 
 The test Job and the mock application deployment are in the `test` Namespace. 
 
 ![Application Gateway tests architecture](./assets/app-gateway-tests-architecture.svg)
 
+## Mock application
+
+Mock application exposes the following APIs:
+- endpoints implementing various authentication methods and returning tokens exposed on 8080 port
+- endpoints implementing mTLS authentication method and returning OAuth tokens exposed on 8090 port
+- endpoints implementing mTLS authentication method and returning OAuth tokens exposed on 8091 port
```

Comment:
I'd add here that it uses expired certs (that's the difference between `8090` and `8091` ports)